db/migrate/20230713150940_add_language_index_to_articles.rb

```diff
@@ -0,0 +1,6 @@
+class AddLanguageIndexToArticles < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+  def change
+    add_index :articles, :language, algorithm: :concurrently
```

Comment:
I proactively added an index here because this is certainly going to be a field which will be queried upon in a variety of ways later.